CMakeLists.txt

```diff
@@ -240,6 +240,9 @@ option(WITH_KVS "Key value store is here" ON)
 # remote block storage
 option(WITH_RBD "Remote block storage is here" ON)
 
+# embedded ceph daemon library
+option(WITH_EMBEDDED "build the embedded ceph daemon library" ON)
```

Comment:
Perhaps a comment here and/or in libcephd.cc would be a good place to clarify the authors' intent of the source code license with respect to linking (IANAL but I am led to understand that authors' understanding and intent is important).  Something like:

Ceph is mostly LGPL (see COPYING), which means that static linking brings with it significant restrictions.  While it is permissible to dynamically link LGPL code with proprietary software (and redistribute the combined work), statically linking LGPL code means that the resulting derivative work must also be LGPL, or the proprietary code be provided in object form so that it can be relinked.  See http://www.gnu.org/licenses/gpl-faq.html#LGPLStaticVsDynamic.